frontend/src/metabase/admin/settings/components/EmbeddingSettings/StaticEmbeddingSettings.tsx

```diff
@@ -34,6 +36,9 @@ export function StaticEmbeddingSettings() {
           </Box>
         )}
       </SettingsSection>
+      <ErrorBoundary>
+        <PLUGIN_CONTENT_TRANSLATION.ContentTranslationConfiguration />
+      </ErrorBoundary>
```

Comment:
would prefer the error boundary inside the component, but I realize you're just moving this around